src/ApiService/ApiService/onefuzzlib/TaskOperations.cs

```diff
@@ -122,7 +120,7 @@ public TaskOperations(ILogTracer log, IOnefuzzContext context)
             return;
         }
 
-        _logTracer.Error($"task failed {task.JobId}:{task.TaskId} - {error}");
+        _logTracer.Info($"task failed {task.JobId}:{task.TaskId} - {error}");
```

Comment:
I guess you're doing this because we're intentionally marking a task as failed?

Reply:
yes, it's not a bug, it's normal operation